docs/components/morphotagger.rst

```diff
@@ -426,7 +442,7 @@ Model parameters are:
 - `char_conv_layers` - number of convolution layers applied to character embeddings (default=1)
 - `char_window_size` - width of convolution filters (default=5). It can be a list if several parallel filters
```

Comment:
I strongly encourage you to replace this long description with simple reference to class docstring in API reference. It is much easier to maintain the single source of truth in case of future changes.